tests/visual-regression/specs/visual-snapshots.test.js

```diff
@@ -0,0 +1,189 @@
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+const screenshotOptions = {
+	fullPage: true,
+};
+
+async function hideDynamicElements( elements ) {
+	for ( let i = 0; i < elements.length; i++ ) {
+		const elementOnPage = await page.$( elements[ i ] );
+		if ( elementOnPage ) {
+			await page.$eval( elements[ i ], ( el ) => {
+				el.style.visibility = 'hidden';
+			} );
+		}
+	}
+	await new Promise( ( resolve ) => setTimeout( resolve, 1000 ) );
```

Comment:
The alternative would be `page.waitForTimeout( 1000 )`:
https://github.com/puppeteer/puppeteer/blob/v5.5.0/docs/api.md#pagewaitfortimeoutmilliseconds

But was wondering why it's needed?

Reply:
The styling changes take a little time to apply, so without the timeout, the screenshot would be taken before the elements were actually hidden.

Reply:
Hmm, `page.waitForTimeout` is too new an API for our version of puppeteer 😬

Reply:
Ah, it might just be `page.waitFor` in this version. They deprecated that more recently in favour of separate `waitForTimeout` and `waitForSelector` functions.